drivers/hmis/app/models/hmis/hud/validators/organization_validator.rb

```diff
@@ -0,0 +1,9 @@
+class Hmis::Hud::Validators::OrganizationValidator < Hmis::Hud::Validators::BaseValidator
+  IGNORED = [
+    :ExportID,
```

Comment:
Why is this mountain case?

Reply:
This is ignoring things in the structure files, which are mountain case and I'm not sure they have aliases like the models do.